build/dockerfiles/rhel.Dockerfile

```diff
@@ -52,6 +52,16 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+
+FROM registry.access.redhat.com/ubi8/nodejs-12:1-102 as dwtemplates
+USER 0
+
+RUN npm install -g @eclipse-che/che-theia-devworkspace-handler
```

Comment:
I think it won't work in brew environment (cc @nickboldt for downstreaming)

Reply:
Wouldn't it be overridden here: https://github.com/redhat-developer/codeready-workspaces/blob/crw-2-rhel-8/dependencies/che-devfile-registry/build/dockerfiles/Dockerfile?

Reply:
yes, CRW devfile registry is a fork. There's no sync between che devfile reg and the crw one.

Therefore you need to manually apply these changes to downstream. But you cannot just `npm install` in brew unless either:
* you prebuild the bits in jenkins as a tarball then unpack them during the brew build as if doing an offline build, or
* use cachito to fetch npm dependencies https://issues.redhat.com/browse/CRW-2444